data/random-teams.ts

```diff
@@ -102,6 +102,10 @@ const Hazards = [
 const ProtectMove = [
 	'banefulbunker', 'protect', 'spikyshield',
 ];
+// Moves that switch the user out
+const PivotingMoves = [
```

Comment:
`camelCase`

Reply:
I think these top-level constants could just as well be `SHOUTING_SNAKE_CASE`?

Reply:
i agree with annika

Reply:
we'll make these adjustments in the gen 9 file for gen 9's next current-gen set update, alongside the arrayification of the move incompatibility list and some set updates. This update will occur this weekend.

Reply:
resolving because i have confirmed that annika does not want this to happen in this pr